src/main/java/seedu/dengue/model/range/EndAge.java

```diff
@@ -8,6 +8,8 @@
  */
 public class EndAge extends Age implements End<Age> {
 
+    public static final Age DEFAULT_ENDAGE = new Age("200");
```

Comment:
Maybe add an extra static variable to avoid magic numbers?

```suggestion
    private static final String MAX_AGE = "200";
    public static final Age DEFAULT_ENDAGE = new Age(MAX_AGE);
```

Reply:
End up im not even using this whoops